Core/GDCore/Extensions/Builtin/MathematicalToolsExtension.cpp

```diff
@@ -18,13 +18,63 @@ BuiltinExtensionsImplementer::ImplementsMathematicalToolsExtension(
       "A set of mathematical functions that can be used in expressions.",
       "Florian Rival",
       "Open source (MIT License)");
-
+  extension.AddInstructionOrExpressionGroupMetadata(_("Mathematical tools"))
+      .SetIcon("res/mathfunction.png");
+
+  extension
+      .AddExpression("Random",
+                     _("Random integer"),
+                     _("Random integer"),
+                     _("Random"),
+                     "res/actions/position.png")
+      .SetHelpPath("/all-features/expressions")
+      .AddParameter("expression", _("Maximum value"));
+
+  extension
+      .AddExpression("RandomInRange",
+                     _("Random integer in range"),
+                     _("Random integer in range"),
+                     _("Random"),
+                     "res/actions/position.png")
+      .SetHelpPath("/all-features/expressions")
+      .AddParameter("expression", _("Minimum value"))
+      .AddParameter("expression", _("Maximum value"));
+
+  extension
+      .AddExpression("RandomFloat",
+                     _("Random float"),
+                     _("Random float"),
+                     _("Random"),
+                     "res/actions/position.png")
+      .SetHelpPath("/all-features/expressions")
+      .AddParameter("expression", _("Maximum value"));
+
+  extension
+      .AddExpression("RandomFloatInRange",
+                     _("Random float in range"),
+                     _("Random float in range"),
+                     _("Random"),
+                     "res/actions/position.png")
+      .SetHelpPath("/all-features/expressions")
+      .AddParameter("expression", _("Minimum value"))
+      .AddParameter("expression", _("Maximum value"));
+
+  extension
+      .AddExpression("RandomWithStep",
+                     _("Random value in steps"),
+                     _("Random value in steps"),
+                     _("Random"),
```

Comment:
do we need a category for those expressions, considering the others don't have one?

Reply:
Removing for consistency!